cart-page.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This app is platform where you can sell or buy stuff">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">  
+    <link rel="stylesheet" href="styles/cart-page.css" type="text/css">
+    <title>GITShop</title>
+</head>
+<body>
+    <nav class="nav-page">
+        <ul class="list-page">
+            <li> GITShop</li>
+            <li><a href="index.html" id="return-home">CONTINUE SHOPPING</a></li>
+
+        </ul>
+    </nav>
+    <section class="sec-1">
+    <div class="section-1">
+        <h3 class="h3-pro">Shopping Bag</h3>
+        <span class="span-sh">Prices are subject to change based on the price in effect the day you checkout</span>
+        <table class="tab-pro">
```

Comment:
we should have 2 way to show result ( list and grid ) ....so we have to do this in js (dom file )

Reply:
done